src/buffer.rs

```diff
@@ -226,7 +226,7 @@ impl<'a, T> Outputs<'a, T> {
     }
 
     /// Mutably access channel at the given index, unchecked
```

Comment:
I'm not sure why the documentation states "unchecked".

Reply:
Indeed. With the types in place, there doesn't seem to be anything unchecked about this. Giving an out-of-bounds index will panic, as it should.

Same thing goes for `get` in `Inputs`.